### PR TITLE
feat: add specific vars for pax ambience

### PR DESCRIPTION
**Pull request by sidnov**
Fixes #[issue_no]

## Summary of Changes

Request from @hotshotp for general ambience vars that indicate passenger presence on board the aircraft. Also fixes the nomenclature for sound-based LVars..

## Additional context

Discord username (if different from GitHub): sidnov# 8337

## Testing instructions

Not required, will be tested as part of the sound PR.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by hotshotp — APPROVED** [review]
Lgtm 😻

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js [hunk 1]
```diff
@@ -131,23 +131,29 @@ class A32NX_Boarding {
...
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_BOARDING", "Bool", false);
         }
 
         if ((currentPax > paxTarget) && boardingStartedByUser == true) {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_DEBOARDING_SOUND`, "Bool", true);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", true);
         } else {
-            await SimVar.SetSimVarValue(`L:A32NX_PAX_DEBOARDING_SOUND`, "Bool", false);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", false);
         }
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js:144]:
```suggestion
        await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_DEBOARDING", "Bool", currentPax > paxTarget && boardingStartedByUser == true);        
```

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js [hunk 2]
```diff
@@ -131,23 +131,29 @@ class A32NX_Boarding {
...
 
         if ((currentPax == paxTarget) && this.isBoarding == true) {
-            await SimVar.SetSimVarValue(`L:A32NX_BOARDING_COMPLETE_SOUND`, "Bool", true);
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_BOARDING_COMPLETE", "Bool", true);
             this.isBoarding = false;
             return;
-        } await SimVar.SetSimVarValue(`L:A32NX_BOARDING_COMPLETE_SOUND`, "Bool", false);
+        } await SimVar.SetSimVarValue("L:A32NX_SOUND_BOARDING_COMPLETE", "Bool", false);
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js:150]:
```suggestion
        }
        await SimVar.SetSimVarValue("L:A32NX_SOUND_BOARDING_COMPLETE", "Bool", false);
```

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js [hunk 3]
```diff
@@ -131,23 +131,29 @@ class A32NX_Boarding {
...
+
+        if (currentPax > 0) {
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_AMBIENCE", "Bool", true);
+        } else {
+            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_AMBIENCE", "Bool", false);
+        }
```
Comment (davidwalschots) [inline flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js:156]:
```suggestion
        await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_AMBIENCE", "Bool", currentPax > 0);
```

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js (hunk 1 above)
Comment (sidnov) [reply flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js]: learnt something new, thanks for the suggestions, implemented 🎉

**Pull request merged**